### DISCUSSION TITLE
Mac M1 support for iOS app

### DISCUSSION THREAD

**Discussion by hey-its-slowly** [phantom/docs#107] [General]
Since the app store supports iOS and iPad OS installations for Apple Silicon macs, are there any plans to support this so we can use Phantom Wallet for iOS on M1? 

The reason I ask is because Phantom Wallet for Chrome does not appear to support Bluetooth for Ledger Nano X or other Bluetooth-enabled hardware wallets.

**Comment by adamdelphantom** [comment]
@hey-its-slowly Thank you for writing in!

There are no immediate plans to implement this. My recommendation would be to submit a feature request for either or both of these features to our Canny board [here](https://phantom.canny.io).
